### PR TITLE
Natalie/fix who is oncall

### PR DESCRIPTION
**Pull request by Mnau-mnau**
custom on call

### CODE REVIEW

File: src/scripts/pagerduty-custom.coffee [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "schedule_ids[]": ['PDHLWLB' , 'PGWSC3H', 'PGFERYM'],
+    since: timeQuery.since
+    until: timeQuery.untilParam
+  }
+
+  pagerduty.get('/oncalls', query, (err, json) ->
+    if err
+      msg.send(err)
+
+    userSupports = json.oncalls.filter((oncall) -> oncall.schedule.id is 'PDHLWLB')
```
Comment (abtris) [inline src/scripts/pagerduty-custom.coffee]: Wasn't this as environment variables (`PDHLWLB`) ?

File: src/scripts/pagerduty-custom.coffee [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  if not msg?
+    console.log('no msg sent')
+    return
+
+  timeNow = Date.now()
+  timeQuery = setTimeQuery(timeFrame, timeNow)
+  query = {
+    limit: 50
+    time_zone: 'UTC'
+    "schedule_ids[]": ['PDHLWLB' , 'PGWSC3H', 'PGFERYM'],
```
Comment (abtris) [inline src/scripts/pagerduty-custom.coffee]: The same issue here, we used env variables for this.

**Review by abtris — CHANGES_REQUESTED** [review]

**Comment by abtris** [comment]
@Mnau-mnau I see only issue in hardcoded values with PD ID's

**Comment by Mnau-mnau** [comment]
I have split the variables, I guess it would have to be defined somewhere where we run Jarvis as well

**Review by abtris — APPROVED** [review]

**Pull request merged**